features/commands/exit_statuses.feature

```diff
@@ -0,0 +1,136 @@
+Feature: Check exit status of commands
```

Comment:
Yes, exactly! - it's documenting how Aruba behaves, rather than just testing it by exercising Aruba features.